app/src/main/java/swati4star/createpdf/fragment/SplitFilesFragment.java

```diff
@@ -130,6 +130,10 @@ public void parse() {
         ArrayList<String> outputFilePaths = mPDFUtils.splitPDFByConfig(mPath,
                 mSplitConfitEditText.getText().toString());
         int numberOfPages = outputFilePaths.size();
+        if (numberOfPages == 1) {
+            showSnackbar(getActivity(), "Selected PDF can\'t be split because it has only 1 page!");
```

Comment:
Store this message as a string resource so that it can be translated for other languages.

Reply:
Done :heavy_check_mark: